resources/ext/ezbake.conf

```diff
@@ -7,8 +7,7 @@
 ezbake: {
    pe: {}
    foss: {
-      redhat: { dependencies: ["puppet >= 3.7.3"
-                               "puppet < 4.0.0"
+      redhat: { dependencies: ["puppet-agent >= 0.2.0"
```

Comment:
puppet-agent should just work. we may want to add a ceiling here at some point to insulate against breaking changes in the agent.